.github/workflows/ci-backend-hbase.yml

```diff
@@ -0,0 +1,89 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: CI Backend Hbase
+
+on:
+  pull_request:
+    paths-ignore:
+    - 'docs/**'
+    - '.github/workflows/ci-docs.yml'
+    - 'requirements.txt'
+    - 'docs.Dockerfile'
+    - '*.md'
+  push:
+    paths-ignore:
+    - 'docs/**'
+    - '.github/workflows/ci-docs.yml'
+    - 'requirements.txt'
+    - 'docs.Dockerfile'
+    - '*.md'
+
+env:
+  BUILD_MAVEN_OPTS: "-DskipTests=true --batch-mode --also-make"
+  VERIFY_MAVEN_OPTS: "-Pcoverage"
+
+jobs:
+  build-all:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - run: mvn clean install --projects janusgraph-all -Pjanusgraph-cache -Dmaven.javadoc.skip=true ${{ env.BUILD_MAVEN_OPTS }}
+      - run: mvn verify --projects janusgraph-all -Pjanusgraph-cache ${{ env.VERIFY_MAVEN_OPTS }}
+
+  tests:
+    runs-on: ubuntu-20.04
+    needs: build-all
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - module: hbase-parent/janusgraph-hbase-10
+            args: "-Dtest=\"**/diskstorage/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            args: "-Dtest=\"**/graphdb/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            args: "-Dtest=\"**/hadoop/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            install-args: "-Dhbase.profile -Phbase2"
+            args: "-Dtest=\"**/diskstorage/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            install-args: "-Dhbase.profile -Phbase2"
+            args: "-Dtest=\"**/graphdb/hbase/*\""
+          - module: hbase-parent/janusgraph-hbase-10
+            install-args: "-Dhbase.profile -Phbase2"
+            args: "-Dtest=\"**/hadoop/*\""
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - run: mvn clean install --projects janusgraph-${{ matrix.module }} ${{ env.BUILD_MAVEN_OPTS }} ${{ matrix.install-args }}
```

Comment:
In TravisCI we also have arguments `-Dmaven.javadoc.skip=true --show-version`

Reply:
I'm not sure what this brings. I will look up the introduction commit

Reply:
I don't know if we need these options. I just noticed that we used them before.